kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -37,66 +37,37 @@ define([
     'common/jobs',
     'services/kernels/comm',
     'common/semaphore',
-], (Promise, devMode, Utils, Jupyter, Runtime, Jobs, JupyterComm, Semaphore) => {
+    'json!kbase/config/job_config.json',
+], (Promise, devMode, Utils, Jupyter, Runtime, Jobs, JupyterComm, Semaphore, JobConfig) => {
     'use strict';
 
     const COMM_NAME = 'KBaseJobs',
+        // channel types
         CELL = 'cell',
         JOB = 'jobId',
-        // frontend params
+        CHANNELS = {
+            CELL,
+            JOB,
+        },
+        // backend param names
+        BE_PARAMS = JobConfig.params,
+        // frontend param names
```

Comment:
the BE and FE job params will be synced in a PR in the near future. Hurrah!

Reply:
Not necessary now, but we can also just update the narrativeConfig module to take in `job_config.json` as well.

There's a module that needs a little update, too. Some future work, maybe a github issue?